runtime.md

```diff
@@ -90,18 +94,27 @@ Attempting to start an already started container MUST have no effect on the cont
 `run <container-id>`
 
 This operation MUST generate an error if it is not provided the container ID.
+Attempting to run a container that does not exist MUST generate an error.
 This operation MUST invoke the `create` operation, and if there are no errors, then invoke the `start` operation.
 The implementation MAY immediately execute `start` without any pause after `create` completes.
 For example if the implementation used a temporary process to implement `create`, it need not do this in the implementation of `run`.
 
+### Stop
+`stop <container-id>`
+
+This operation MUST generate an error if it is not provided the container ID.
+Attempting to stop a container that is not running MUST have no effect on the container and MUST generate an error.
+This operation MUST stop all of the processes running within the scope of the container.
+This operation MUST NOT delete any resources associated with the container, except for the PID namespace.
```

Comment:
On Mon, Apr 25, 2016 at 10:46:19AM -0700, Doug Davis wrote:
“The concern I have is that if we just split create/start and not
stop/delete then we're only doing 1/2 a solution because hooks will
still need to be there…”

I think one of the things missed in earlier discussions was that the
post-stop hooks currently happen after container cleanup (see #395
which wants to change this).  So I think we can drop all hooks in this
PR without splitting stop/delete.  If #395 gets traction, it can
always add post-stop hooks back in where it wants them later (or split
stop/delete, or whatever).

“I'd actually prefer it so that we don't have to revisit this PR's
decisions when we do get around to the stop/delete discussions.”

What decisions would we have to revisit?  Putting post-stop hooks back
in (or splitting stop/delete) doesn't seem like it would have many
knock-on effects for the create/start split.  On the other hand,
Wednesday isn't far off.  Maybe we can get a stronger consensus on the
stop/delete split than we got during the 2016-03-30 meeting, when we
basically beat on this single issue for 50 minutes [1] ;).

[1]: http://ircbot.wl.linuxfoundation.org/meetings/opencontainers/2016/opencontainers.2016-03-30-17.01.log.html#l-13